idea_town/frontend/static-src/app/main.js

```diff
@@ -7,12 +8,41 @@ import Me from './models/me';
 import PageManager from './lib/page-manager';
 import Router from './lib/router';
 
+function getDocHeadLink(relName, attrName) {
+  return document
+    .querySelector('head link[rel="' + relName + '"]')
+    .getAttribute(attrName);
+}
+
 app.extend({
+
   initialize() {
+
     app.experiments = new ExperimentsCollection();
     app.experiments.fetch();
```

Comment:
Just pushed a commit to do that... 

I was also thinking with issue #83 we could use fetch & promises to more easily batch up many HTTP requests, if we end up needing to do more than these two that at some point.